src/static/js/modules/loan-comparison/mortgage-calculations.js

```diff
@@ -56,12 +64,12 @@ mortgage['monthly-hoa-dues'] = function (loan) {
 };
 
 mortgage['monthly-principal-interest'] = function (loan) {
-    return amortize({
-      amount: positive(loan['loan-amount']),
-      rate: loan['interest-rate'],
-      totalTerm: loan['loan-term'] * 12,
-      amortizeTerm: 60 // @todo loan term * 12?
-    }).payment;
+    return Math.round(amortize({
+                  amount: positive(loan['loan-amount']),
+                  rate: loan['interest-rate'],
+                  totalTerm: loan['loan-term'] * 12,
+                  amortizeTerm: 60 // @todo loan term * 12?
```

Comment:
@cfarm I checked with @contolini, amortize term is the number of payment you have already made for the loan.  So I am assuming we are doing new loan here, so amortize term would be 0.